Makefile

```diff
@@ -18,12 +18,14 @@ test:
 
 .PHONY: cluster-prepare
 cluster-prepare:
+	$(MAKE) -C third_party/istio deploy
```

Comment:
As mentioned in the issue, let's wait with adding it to cluster prepare until this becomes a strong dependency (i.e., when we merge tests for control plane security)

Reply:
ok